MHTimer.js

```diff
@@ -201,6 +202,11 @@ function Main() {
                 .then(getMouseList)
                 .then(getItemList)
                 .then(getFilterList);
+            resetRH();
+            // But then get starting values with notifying people just because of a restart
+            (async function() {
+                await getRHLocation();
```

Comment:
OK, we don't care that it's asynchronous because it should finish soon enough that it'll be done before it causes any real headaches such as someone doing a quick find of rh right before midnight which somehow gets asynchronously completed before the startup getRHLocation which runs right after midnight so gets the new location. Or some weirdness there.

Seems fairly safe!

Reply:
This remoteData chain is required to complete before the bot attempts to log in (or at least that's the intent. It might not actually strictly require that at the moment, but it does initiatiate all of these requests before attempting to log in).